### PR TITLE
Fix/deliberations fixes

### PR DESCRIPTION
**Pull request by vitorvicente**
Fixed two major issues with the deliberations:
- Prior data in the currentApplicationDeliberation localStorage would not have a match and crash
- When someone else updated an application the entire thing would fall back to the first application because we were storing the full application and not the UID

### CODE REVIEW

File: src/components/Deliberation/InterviewerView.js [hunk 1]
```diff
@@ -94,7 +94,8 @@ class InterviewerView extends Component {
   _initFirebase = false;
   state = {
     applications: [], // TODO: instead of loading all the applications by default, load their ids, then pass that id to application display/admin settings so they can handle accordingly
```
Comment (ROODAY) [inline src/components/Deliberation/InterviewerView.js:96]: Can we remove this TODO now?

File: src/components/Deliberation/InterviewerView.js [hunk 2]
```diff
@@ -199,17 +207,23 @@ class InterviewerView extends Component {
...
       questions,
       levelConfig,
-      currentApplication,
+      currentApplicationID,
       loading: false,
     });
   };
 
   setCurrentApplication = (currentApplication) => {
+    console.log(currentApplication);
```
Comment (ROODAY) [inline src/components/Deliberation/InterviewerView.js:216]: Let's get rid of this console.log if we're done testing

**Review by ROODAY — CHANGES_REQUESTED** [review]
Looks good overall, just those 2 comments I left earlier about the TODO and console.log. Might also be a good time to put some comments throughout the code, as now that I'm looking back at it a few months later it's hard to follow along, I can only imagine how hard it'd be for someone brand new to the codebase.

**Comment by vitorvicente** [comment]
Yeah it was hard enough for me to figure out why we were storing the entire application in localhost instead of the uid as is, so I think I'm gonna just comment a lot of stuff everywhere in a different PR

I'll fix these two things up though

**Review by ROODAY — APPROVED** [review]
